docs/event-types.mdx

```diff
@@ -432,6 +432,41 @@ done < data.csv
 </TabItem>
 </CodeTabs>
 
+### Upload OpenAPI specification
```

Comment:
Looks good overall, but please ask someone from the team (usually James like doing it) for some proofreading/rewording help.